definitions/scenarios/upgrade_to_satellite_6_10.rb

```diff
@@ -26,8 +26,12 @@ class PreUpgradeCheck < Abstract
 
     def compose
       add_step(Checks::Puppet::WarnAboutPuppetRemoval)
+      add_step(Checks::CheckForNewerPackages.new(:packages => ['tfm-rubygem-katello',
```

Comment:
I dont think this will be executed on EL8, however there is new method added `foreman_plugin_name` in the system_helpers which can be used to take care of scl/nonscl package naming: https://github.com/theforeman/foreman_maintain/blob/master/lib/foreman_maintain/concerns/system_helpers.rb#L222